README.md

```
@@ -633,6 +634,16 @@ This preset loads confetti moving horizontally across the screen like a parade.
 
 You can find the instructions [here](https://github.com/tsparticles/presets/tree/main/presets/confettiParade#readme)
 
+### Party
+
+[![jsDelivr](https://data.jsdelivr.com/v1/package/npm/@tsparticles/preset-party/badge)](https://www.jsdelivr.com/package/npm/@tsparticles/preset-party) [![npmjs](https://badge.fury.io/js/@tsparticles/preset-party.svg)](https://www.npmjs.com/package/@tsparticles/preset-party) [![npmjs](https://img.shields.io/npm/dt/@tsparticles/preset-party)](https://www.npmjs.com/package/@tsparticles/preset-party)
+
+This preset creates an animated confetti-like celebration effect.
+
+[![demo](https://github.com/tsparticles/presets/raw/main/presets/party/images/sample.png)](https://particles.js.org/samples/presets/party)
+
+You can find the instructions [here](https://github.com/tsparticles/presets/tree/main/presets/party#readme)
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)